### PR TITLE
Rollbar integration

### LINKED ISSUE #537: Rollbar integration for AngularJS
Library to use: https://github.com/tandibar/ng-rollbar
Access Token: `d0a14094307a424cbd7d59d9c897b940`

* Give ng-rollbar project a star
* take care of staging and production env vars
* Disable in Localhost

### PR DESCRIPTION
**Pull request by ArsalanSavand**
# Bug fixed

Fixes #537

### CODE REVIEW

File: app/app_config.js [hunk 1]
```diff
@@ -41,6 +43,22 @@ function Config($httpProvider, $resourceProvider, $cookiesProvider, $qProvider,
...
+
+  // Check localhost
+  var localhost = (
+    window.location.href.indexOf("localhost") > 0 || window.location.href.indexOf("127.0.0.1") > 0
+  );
+  
+  // Rollbar integration
+  if (!localhost) {
+    RollbarProvider.init({
+      accessToken: "d0a14094307a424cbd7d59d9c897b940",
```
Comment (AmirSavand) [inline app/app_config.js]: Need to load it from ENV.ROLLERBAR_TOKEN

File: app/app_config.js [hunk 2]
```diff
@@ -41,6 +43,22 @@ function Config($httpProvider, $resourceProvider, $cookiesProvider, $qProvider,
       }
     }
   });
+
+  // Check localhost
+  var localhost = (
+    window.location.href.indexOf("localhost") > 0 || window.location.href.indexOf("127.0.0.1") > 0
```
Comment (AmirSavand) [inline app/app_config.js]: Should check based on `ENV.name` not URL.

Comment (ArsalanSavand) [inline app/app_config.js]: Amir i know but Alireza said this is the right way.
Staging is still local host.

File: app/app_config.js (hunk 1 above)
Comment (ArsalanSavand) [inline app/app_config.js]: Do I have to add it manually?

Comment (Alir3z4) [inline app/app_config.js]: Yeah, this should be in ENV

File: app/app_config.js (hunk 2 above)
Comment (Alir3z4) [inline app/app_config.js]: Yes, this is the right way.
Let's keep it this way

**Comment by AmirSavand** [comment]
@ArsalanSavand finish this up.

File: app/app_config.js (hunk 1 above)
Comment (ArsalanSavand) [inline app/app_config.js]: Done

**Comment by ArsalanSavand** [comment]
Done.

**Pull request merged**